assets/sass/pages/uses.scss

```diff
@@ -0,0 +1,221 @@
+.uses-page {
+  position: relative;
+
+  .hero-background {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: linear-gradient(to right, #1E85E3, #23B9FC);
+    width: 100%;
+    height: 0;
+    padding-bottom: 50%;
+    display: none;
+
+    @media only screen and (min-width: 620px) {
+      display: block;
+    }
+
+    &::before {
+      content: '';
+      position: absolute;
+      top: 200px;
+      left: -60px;
+      background: radial-gradient(circle, #1E85E3, #1E85E3, #23B9FC);
+      width: 24%;
+      height: 8px;
+      transform: rotate(-35deg);
+
+      @media only screen and (min-width: 620px) and (max-width: 800px){
+        top: 80px;
+        left: -26px;
+        transform: rotate(-26deg);
+      }
+
+      @media only screen and (min-width: 801px) and (max-width: 1100px){
+        top: 120px;
+        left: -26px;
+        transform: rotate(-26deg);
+      }
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      top: 300px;
+      left: -60px;
+      background: linear-gradient(to left, #237ABD, #1A89F4);
+      width: 20%;
+      height: 8px;
+      transform: rotate(-35deg);
+
+      @media only screen and (min-width: 620px) and (max-width: 800px){
+        top: 120px;
+        left: -26px;
+        transform: rotate(-26deg);
+      }
+
+      @media only screen and (min-width: 801px) and (max-width: 1100px){
+        top: 180px;
+        left: -20px;
+        transform: rotate(-26deg);
+      }
+    }
+  }
+
+  .hero-background-white {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 0 0 0 0;
+    border-color: transparent transparent #ffffff transparent;
+    display: block;
+
+    @media only screen and (min-width: 620px) {
+      display: block;
+    }
+
+    @media only screen and (min-width: 620px) and (max-width: 749px){
+      border-width: 0 0 50vw 200vw;
+      left: -35px;
+    }
+
+    @media only screen and (min-width: 750px) and (max-width: 1299px){
+      border-width: 0 0 50vw 150vw;
+      left: -25px;
+    }
+
+    @media only screen and (min-width: 1300px) {
+      border-width: 0 0 50vw 100vw;
+      left: 0;
+    }
+  }
+
+  .hero {
+    max-width: none;
+    margin: auto;
+    text-align: center;
+    padding: 40px 20px;
+    position: relative;
+    background: linear-gradient(to right, #1E85E3, #23B9FC);
+
+    @media only screen and (min-width: 620px) {
+      background: transparent;
+      max-width: 500px;
+    }
+  }
+
+  .hero-title {
+    color: #ffffff;
+  }
+
+  .hero-description {
```

Comment:
I'm finding the description is rather faint.

Reply:
I'm wondering if part of the reason is because the GIF preview is low res. Here is a more clear png.
![light](https://user-images.githubusercontent.com/25016368/35995484-5880f4e6-0cc8-11e8-96f8-16c61c815e6a.png)

We could potentially change the font to regular instead of light. That would look like this: 
![regular](https://user-images.githubusercontent.com/25016368/35995501-681a9df8-0cc8-11e8-81fb-0711abc468c8.png)

Reply:
I actually ran it locally and it seemed faint on my desktop (Linux)

Reply:
If you think it's necessary we can change the font from light to regular.

Reply:
Looks fine on my Mac.  Leave as is. TY.